airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml

```diff
@@ -67,7 +67,7 @@ definitions:
       datetime: "{{ config['start_time'] }}"
       datetime_format: "%s"
     end_datetime:
-      datetime: "{{ now_utc() }}}"
```

Comment:
fixes time parsing that would normally fail when it gets an unexpected `}`

Reply:
Did this cause any test failures?

Reply:
It didn't, but i think the reason was because the `messages` stream uses an API endpoint that our test creds don't have access to which then returns a 400 error code. I haven't dug into how our unit tests from a month or two ago are working, but its possible they see the same error and move on to the next set of streams